test/e2e/end_user_test.go

```diff
@@ -23,6 +23,92 @@ import (
 	"k8s.io/apimachinery/pkg/util/wait"
 )
 
+var _ = Describe("Openshift on Azure end user tests [AdvancedEndUser]", func() {
```

Comment:
Why is this advanced? It should be something end-users should be able to run. I don't see any gain for the separate suite.